offchain/components/relayer/src/includer.rs

```diff
@@ -106,7 +106,8 @@ impl SolanaIncluder {
                 message = receiver.recv() => {
                     trace!("received a new message");
                     let approval = message.ok_or(IncluderError::ChannelClosed)?;
-                    futures.push(worker_future(approval));
+                    let worker_clone = worker.clone();
+                    futures.push(async move {worker_clone.include(approval).await});
```

Comment:
this should work too
```suggestion
                    futures.push(worker.clone().include(approval));
```